_tests/integration/android_test.go

```diff
@@ -201,22 +316,77 @@ var sampleAppsAndroid22Versions = []interface{}{
 	steps.CachePullVersion,
 	steps.ScriptVersion,
 	steps.InstallMissingAndroidToolsVersion,
-	steps.GradleRunnerVersion,
+	steps.AndroidLintVersion,
+	steps.AndroidUnitTestVersion,
 	steps.DeployToBitriseIoVersion,
 	steps.CachePushVersion,
 }
 
 var sampleAppsAndroid22ResultYML = fmt.Sprintf(`options:
   android:
-    title: Path to the gradle file to use
-    env_key: GRADLE_BUILD_FILE_PATH
+    title: The root directory of an Android project
```

Comment:
it seems that this test tests the same as `sampleAppsAndroidSDK22SubdirResultYML` at least the generated config seems to be the same, do we need this test as well?

Reply:
It is not the same. `sampleAppsAndroidSDK22SubdirResultYML` has 'src' as project location option, and this one has `.` as project location